.travis.yml

```diff
@@ -1,17 +1,35 @@
 language: python
-python:
-  - "2.6"
-  - "2.7"
-  - "3.2"
-  - "3.3"
-  - "3.4"
-  - "pypy"
-script: make test
+script: tox
 before_install:
   - openssl version
+  # Install Google Cloud Platform SDK to enable App Engine tests
+  - if [[ $TOXENV == 'gae' && ! -d $HOME/gcloud/google-cloud-sdk ]]; then
+      mkdir -p $HOME/gcloud &&
+      wget https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz --directory-prefix=$HOME/gcloud &&
```

Comment:
Can we pin the sdk to a specific release? We do it with all other dependencies too.
Best store the version in an env variable?

We could also move the download to tox. Which would probably annoy developers, but we could then exclude gae from the default envlist. @shazow?